bin/plugin/commands/changelog.js

```diff
@@ -97,6 +99,21 @@ const LABEL_TYPE_MAPPING = {
 	'[Type] Security': 'Security',
 };
 
+const LABEL_FEATURE_MAPPING = {
+	'[Package] Block Editor': 'Block Editor',
+	'[Package] Editor': 'Editor',
+	'[Package] Edit Widgets': 'Widgets',
+	'[Package] Widgets Customizer': 'Widgets',
+	'[Feature] Widgets Screen': 'Widgets',
+	'[Block] Legacy Widget': 'Widgets',
```

Comment:
Since the project is usually referred to as "Widgets Editor" I would go with that name here as well.